imod/mf6/ssm.py

```diff
@@ -146,7 +146,7 @@ def from_flow_model(
                 aux_var_names.append(species)
 
         if len(names) == 0:
-            raise ValueError("flow model does not contain boundary conditions")
```

Comment:
Is this appropriate behavior? Or required?
We can log a warning?

Reply:
before we implemented partitioning, this was a red flag. But now, we can have a partition which is not on the boundary, and it will not have flow boundaries,  so it is not a red flag anymore.